src/main/java/com/google/cloud/genomics/utils/grpc/VariantUtils.java

```diff
@@ -60,6 +168,65 @@ public Integer apply(String string) {
       });
 
   /**
+   * Determines whether the variant is filtered according to the FILTER field,
+   * using the notation required by the VCF 4.2 standard. This means that the
+   * FILTER field of the variant must either be empty, or contain only "PASS"
+   * or ".".
+   */
+  public static final Predicate<Variant> IS_FILTERED = new Predicate<Variant>() {
+    @Override
+    public boolean apply(Variant variant) {
+      return variant.getFilterCount() != 0 && !((variant.getFilterCount() == 1)
+          && (variant.getFilter(0).equalsIgnoreCase(PASSES_FILTERS) ||
+          variant.getFilter(0).equalsIgnoreCase(MISSING_VALUE)));
+    }
+  };
+
+  /**
+   * Determines whether a variant has one and only one alternate allele.
+   */
+  public static final Predicate<Variant> IS_BIALLELIC = Predicates.and(HAS_ALTERNATE,
+      new Predicate<Variant>() {
+        @Override
+        public boolean apply(Variant variant) {
+          return variant.getAlternateBasesList().size() == 1;
+        }
+      }
+  );
+
+  /**
+   * Determines whether a variant is a gVCF record.
+   */
+  public static final Predicate<Variant> IS_GVCF = new Predicate<Variant>() {
```

Comment:
I think it is meant only to pass a test since it's not really a valid variant.  The code above just returns a string since it is defined as `public String getAlternateBases(int index) {...}`.  In the [VariantUtilsJava.java](https://github.com/hwangy/utils-java/blob/c0fdfe3daa38d8aaade422d6dec1fd7afc8e5b47/src/test/java/com/google/cloud/genomics/utils/grpc/VariantUtilsTest.java#L325-329) you'll find the following test:

```Java
variant = Variant.newBuilder()
        .setReferenceBases("A")
        .addAlternateBases("<*>")
        .build();
assertTrue(VariantUtils.IS_GVCF.apply(variant));
```